compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -404,14 +404,6 @@ public BSymbol resolveObjectMethod(DiagnosticPos pos, SymbolEnv env, Name fieldN
         return lookupMemberSymbol(pos, objectSymbol.methodScope, env, fieldName, SymTag.VARIABLE);
     }
 
-    public BType resolveTypeNodeWithDeprecationCheck(BLangType typeNode, SymbolEnv env) {
```

Comment:
We may have to deprecate this instead.

Reply:
This was added recently and it had only one usage. I think it is better to delete it.